splitio_android/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.1.7+1-rc.2 (Feb 7, 2023)
```

Comment:
why are you adding `rc2` and without description ?

Reply:
Adding the entries for rc for consistency, and I'll add the full description in the stable release.